build/common.props

```diff
@@ -0,0 +1,23 @@
+<Project>
+  <Import Project="..\version.props" />
```

Comment:
...but, the relative path of Import Project is relative to the file in which its expressed.

Or so I've been led to believe. This works in all of our other repos